mobile/README.md

```diff
@@ -138,6 +138,8 @@ flutter pub run dart_code_metrics:metrics check-unused-code lib
 ### **Building for release**
 
 ```bash
+flutter build appbundle --build-name 1.0.0 --build-number 20025 --flavor airqo
```

Comment:
@NoahNsimbe do we have to change this build number in the README every time we update

Reply:
No. the README is just or reference. We can explore leveraging the above command to build and tag the app with a version without modifying the version in the pubspec file.